cms/static/js/views/pages/container.js

```diff
@@ -909,15 +895,12 @@ function($, _, Backbone, gettext, BasePage,
             this.deleteComponent(this.findXBlockElement(event.target));
         },
 
-        createPlaceholderElement: function() {
```

Comment:
[question]: Will Legacy Studio work correctly without this code?

Reply:
We have the same function on line [L726](https://github.com/openedx/edx-platform/blob/24453ab1a8cce2ae111ca250d2e3056a2289fed4/cms/static/js/views/pages/container.js#L726)

Reply:
Resolved